.github/workflows/firebase-hosting-cleanup.yml

```
@@ -7,12 +7,14 @@ on:
 jobs:
   cleanup:
     runs-on: ubuntu-latest
+    env:
+      FIREBASE_PREVIEW_CHANNEL_ID: pr-${{ github.event.pull_request.number }}
     steps:
-      - uses: actions/checkout@v3.0.2
+      - uses: actions/checkout@v4
       - uses: google-github-actions/auth@v1
```

actionlint (1.7.11)

[error] the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)